### PR TITLE
fix: [menu]File Manager Right Mouse Button Stuck

### PR DESCRIPTION
**Pull request by liyigang1**
Fileinfo was created in the isSuffixSupport function in the dcustomactionbuilder class and the oemmenu class. Modify the selected fileinfo passed in here.

Log: File Manager Right Mouse Button Stuck

### CODE REVIEW

File: src/plugins/common/core/dfmplugin-menu/extendmenuscene/extendmenu/dcustomactionbuilder.cpp [hunk 1]
```diff
@@ -423,19 +423,16 @@ bool DCustomActionBuilder::isSchemeSupport(const DCustomActionEntry &action, con
     return supportList.contains(url.scheme(), Qt::CaseInsensitive);
 }
 
-bool DCustomActionBuilder::isSuffixSupport(const DCustomActionEntry &action, const QUrl &url)
+bool DCustomActionBuilder::isSuffixSupport(const DCustomActionEntry &action, const FileInfoPointer &fileInfo)
```
Comment (max-lvs) [inline src/plugins/common/core/dfmplugin-menu/extendmenuscene/extendmenu/dcustomactionbuilder.cpp]: 智能指针 需要用引用么？

File: src/plugins/common/core/dfmplugin-menu/oemmenuscene/oemmenu.cpp [hunk 2]
```diff
@@ -170,10 +170,9 @@ bool OemMenuPrivate::isSchemeSupport(const QAction *action, const QUrl &url) con
     return supportList.contains(url.scheme(), Qt::CaseInsensitive);
 }
 
-bool OemMenuPrivate::isSuffixSupport(const QAction *action, const QUrl &url, const bool allEx7z) const
+bool OemMenuPrivate::isSuffixSupport(const QAction *action, const FileInfoPointer &fileInfo, const bool allEx7z) const
```
Comment (max-lvs) [inline src/plugins/common/core/dfmplugin-menu/oemmenuscene/oemmenu.cpp]: 智能指针需要引用么

**Review by GongHeng2017 — DISMISSED** [review]

File: src/plugins/common/core/dfmplugin-menu/extendmenuscene/extendmenu/dcustomactionbuilder.cpp (hunk 1 above)
Comment (liyigang1) [reply src/plugins/common/core/dfmplugin-menu/extendmenuscene/extendmenu/dcustomactionbuilder.cpp]: 已处理

**Review by max-lvs — APPROVED** [review]

**Pull request merged**